Tests/MoyaProviderIntegrationTests.swift

```diff
@@ -299,6 +299,30 @@ final class MoyaProviderIntegrationTests: QuickSpec {
                 }
             }
         }
+
+        // Resolves ValidationType not working with multipart upload #1590
+        describe("a provider performing a multipart upload with Alamofire validation") {
+            it("only allows status code 287") {
```

Comment:
how about `it returns error for status code different than 287` and then we could add another case `it returns valid response for status code 287`

Reply:
I wasn't able to test for `287` specifically because I have no control over the actual response since this is not a stub. I just tested for `ValidationType.successCodes` instead